tests/test_config.py

```diff
@@ -0,0 +1,178 @@
+        # 'gn+ws/mask_rcnn_x101_32x4d_fpn_gn_ws_2x.py',
+        'gn+ws/mask_rcnn_r50_fpn_gn_ws_2x.py',
+        # 'gn+ws/mask_rcnn_r50_fpn_gn_ws_20_23_24e.py',
+        # ---
+        # 'wider_face/ssd300_wider_face.py',
+        # ---
+        'pascal_voc/ssd300_voc.py',
+        'pascal_voc/faster_rcnn_r50_fpn_1x_voc0712.py',
+        'pascal_voc/ssd512_voc.py',
+        # ---
+        # 'gcnet/mask_rcnn_r4_gcb_c3-c5_r50_fpn_syncbn_1x.py',
+        # 'gcnet/mask_rcnn_r16_gcb_c3-c5_r50_fpn_syncbn_1x.py',
+        # 'gcnet/mask_rcnn_r4_gcb_c3-c5_r50_fpn_1x.py',
+        # 'gcnet/mask_rcnn_r16_gcb_c3-c5_r50_fpn_1x.py',
+        'gcnet/mask_rcnn_r50_fpn_sbn_1x.py',
+        # ---
+        'gn/mask_rcnn_r50_fpn_gn_contrib_2x.py',
+        # 'gn/mask_rcnn_r50_fpn_gn_2x.py',
+        # 'gn/mask_rcnn_r101_fpn_gn_2x.py',
+        # ---
+        # 'reppoints/reppoints_moment_x101_dcn_fpn_2x.py',
+        'reppoints/reppoints_moment_r50_fpn_2x.py',
+        # 'reppoints/reppoints_moment_x101_dcn_fpn_2x_mt.py',
+        'reppoints/reppoints_partial_minmax_r50_fpn_1x.py',
+        'reppoints/bbox_r50_grid_center_fpn_1x.py',
+        # 'reppoints/reppoints_moment_r101_dcn_fpn_2x.py',
+        # 'reppoints/reppoints_moment_r101_fpn_2x_mt.py',
+        # 'reppoints/reppoints_moment_r50_fpn_2x_mt.py',
+        'reppoints/reppoints_minmax_r50_fpn_1x.py',
+        # 'reppoints/reppoints_moment_r50_fpn_1x.py',
+        # 'reppoints/reppoints_moment_r101_fpn_2x.py',
+        # 'reppoints/reppoints_moment_r101_dcn_fpn_2x_mt.py',
+        'reppoints/bbox_r50_grid_fpn_1x.py',
+        # ---
+        # 'fcos/fcos_mstrain_640_800_x101_64x4d_fpn_gn_2x.py',
+        # 'fcos/fcos_mstrain_640_800_r101_caffe_fpn_gn_2x_4gpu.py',
+        'fcos/fcos_r50_caffe_fpn_gn_1x_4gpu.py',
+        # ---
+        'albu_example/mask_rcnn_r50_fpn_1x.py',
+        # ---
+        'libra_rcnn/libra_faster_rcnn_r50_fpn_1x.py',
+        # 'libra_rcnn/libra_retinanet_r50_fpn_1x.py',
+        # 'libra_rcnn/libra_faster_rcnn_r101_fpn_1x.py',
+        # 'libra_rcnn/libra_faster_rcnn_x101_64x4d_fpn_1x.py',
+        # 'libra_rcnn/libra_fast_rcnn_r50_fpn_1x.py',
+        # ---
+        # 'ghm/retinanet_ghm_r50_fpn_1x.py',
+        # ---
+        # 'fp16/retinanet_r50_fpn_fp16_1x.py',
+        'fp16/mask_rcnn_r50_fpn_fp16_1x.py',
+        'fp16/faster_rcnn_r50_fpn_fp16_1x.py'
+    ]
+
+    print('Using {} config files'.format(len(config_names)))
+
+    times = {}
+    for config_fname in config_names:
+        config_fpath = join(config_dpath, config_fname)
+        config_mod = import_module_from_path(config_fpath)
+
+        config_mod.model
+        config_mod.train_cfg
+        config_mod.test_cfg
+        print('Building detector, config_fpath = {!r}'.format(config_fpath))
+
+        # Remove pretrained keys to allow for testing in an offline environment
+        if 'pretrained' in config_mod.model:
+            config_mod.model['pretrained'] = None
+
+        import ubelt as ub
```

Comment:
Maybe we can avoid involving this library since it is only used here one for testing.
An alternative of the timer is `mmcv.Timer`.

Reply:
Of course. This is actually debug code that I didn't mean to commit.